src/ui/components/shared/CodeEditor/useAutocomplete.tsx

```diff
@@ -118,19 +118,17 @@ function useGetEvalMatches(expression: string) {
   const selectedFrameId = useAppSelector(getSelectedFrameId);
   const replayClient = useContext(ReplayClientContext);
 
-  const pauseId = useAppSelector(getPauseId);
-
   const getEvalMatchesForSelectedFrame = useCallback(
     async (expression: string) => {
-      if (!pauseId) {
+      if (!selectedFrameId) {
         return null;
       }
       const fetchObject = async (objectId: string) => {
-        return getObjectWithPreviewHelper(replayClient, pauseId, objectId);
+        return getObjectWithPreviewHelper(replayClient, selectedFrameId.pauseId, objectId);
```

Comment:
I don't like `selectedFrame` either because it isn't a frame, only the IDs that are necessary to uniquely identify the frame. I've used `PauseAndFrameId` in a couple of places because I couldn't think of a better name.

Reply:
Then call it `selectedPauseAndFrameId` (or just store them separately as `selectedPauseId` and `selectedFrameId`?

Calling it `selectedFrameId` seems really confusing though because it's _not_ a frame id.

Reply:
I've renamed it to `selectedPauseAndFrameId`